### ISSUE TITLE
Proxy mode disconnects the current VPN connection

### ISSUE THREAD

**Issue by Mishasama** [bepass-org/oblivion#435]
Proxy mode should not touch any VPN-related modules, otherwise this feature would be meaningless.

**Comment by PtDragon** [comment]
Yes, i confirm the bug - same happens for me too so the app cannot work as it gets overridden by other "VPN" software. 
I hope the developer will take a moment to fix this(should be simple enough).
